beacon-chain/orchestrator/client.go

```diff
@@ -0,0 +1,57 @@
+package orchestrator
+
+import (
+	"context"
+	"fmt"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+const confirmVanBlockHashesMethod = "orc_confirmVanBlockHashes"
+
+type Client interface {
+	Dial(string) (*Client, error)
+	DialContext(context.Context, string) (*Client, error)
+	NewClient(*rpc.Client) *Client
+	Close()
+	ConfirmVanBlockHashes(context.Context, []*BlockHash) ([]*BlockStatus, error)
```

Comment:
@mxmar Method definition should be `ConfirmVanBlockHashes(ctx context.Context, request []*vanTypes.ConfirmationReqData) (response []*vanTypes.ConfirmationResData, err error)` and in this method, you just convert common.Hash to [32] byte for hash

Reply:
Ok, so conversion logic will lie in this method as you has suggested. I will prepare separate interface for it.

Reply:
done.